sql-cloudant/src/main/scala/org/apache/bahir/cloudant/common/JsonStoreDataAccess.scala

```diff
@@ -125,6 +125,19 @@ class JsonStoreDataAccess (config: CloudantConfig)  {
     val clRequest: HttpRequest = getClRequest(url, postData)
 
     val clResponse: HttpResponse[String] = clRequest.execute()
+          
+    //BAHIR-130 start
+    //In cloudant Lite number of requests pre second are limited.
+    //Just wait and try again
+    clResponse.code match {
+        case 429 => {
+            Thread sleep 1000
```

Comment:
Also, would be good to have logging for:
- When there's too many requests and attempting a retry with x seconds
- Any problems/interrupt during the 'backoff' wait time